deps/thread.lua

```diff
@@ -0,0 +1,464 @@
+--[[
+
+Copyright 2014 The Luvit Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+--]]
+
+--- luvit thread management
+local uv = require('uv')
+local base = require('luvi').bundle.base
+
+exports.name = "luvit/thread"
+exports.version = "0.0.1"
+exports.dependencies = {
+}

```

Comment:
No need for an empty table if there are no dependencies.  The field is optional.